docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -22,21 +22,23 @@
 
     <para>The entire HPCC Systems configuration in the container space, is
     governed by a single file, a <emphasis>values.yaml</emphasis> file, and
-    its associated schema file.</para>
+    its associated schema (<emphasis>values-schema.json</emphasis>)
+    file.</para>
 
     <sect2 id="WhatIsValues.Yaml">
       <title>The <emphasis>values.yaml</emphasis> and how it is used</title>
 
-      <para>The <emphasis>values.yaml</emphasis> file is the delivered
-      configuration values for a Helm chart. The
-      <emphasis>values.yaml</emphasis> file is used by the Helm chart to
-      control how HPCC Systems is deployed to the cloud. This values file is
-      one file used to configure and get an HPCC Systems instance up and
-      running on Kubernetes. The <emphasis>values.yaml</emphasis> file defines
-      everything that happens to configure and/or define your system for a
-      containerized deployment. You should use the values file provided as a
-      basis for modeling the specific customizations for your deployment
-      specific to your requirements.</para>
+      <para>The supplied stock <emphasis>values.yaml</emphasis> file provided
+      in the HPCC Systems repository is the delivered configuration values for
+      the "hpcc" Helm chart. The <emphasis>values.yaml</emphasis> file is used
+      by the Helm chart to control how HPCC Systems is deployed to the cloud.
+      This <emphasis>values.yaml</emphasis> file is a single file used to
+      configure and get an HPCC Systems instance up and running on Kubernetes.
```

Comment:
see previous comment re. an aribitrary number of command line supplied value files, e.g.:
helm install myhpcc hpcc/hpcc -f somevals.yaml -f somemorevals -f espstuff.yaml -f thorstuff.yaml